static/styles/teams/team.scss

```diff
@@ -118,4 +119,12 @@
           padding-right: 48px !important;
       }
   }
-}
\ No newline at end of file
+}
+
+.col-sort-icon {
+	color: $colorTertiary;
+	pointer-events: none;
+	position: fixed;
```

Comment:
i think position fixed is overly specific here. position absolute should suffice